packages/focusvisible/src/FocusVisibleContainer.spec.tsx

```diff
@@ -180,7 +180,7 @@ describe('FocusVisibleContainer', () => {
     it('does not apply focus-visible to input with invalid type', () => {
       const { getByTestId } = render(
         <KeyboardModalityExample>
-          <input data-test-id="input" type="invalid-type" />
+          <input data-test-id="input" type="range" />
```

Comment:
Due to Jest `v26` upgrade, it looks like event objects are now [normalized](https://codesandbox.io/s/react-event-process-vcsyv?file=/src/App.js) (maybe via React?), which was not the case prior to this upgrade. The event type `invalid-type` is now normalized to `text`.  The type `text` is in the [`inputTypesWhitelist`](https://github.com/WICG/focus-visible/blob/master/src/focus-visible.js#L13), so this test fails.

I see a couple of options here:
1) Use an valid input `type` which was [excluded](https://github.com/WICG/focus-visible/pull/218) from the whitelist. I went with `range`.
2) Completely remove this test.
3) TBD.

Reply:
We ran into this with `react-components` as well. I believe it’s from the `jsdom` upgrades. 

I vote that we could remove it or change it to another invalid type.